openhands/agent_server/models.py

```diff
@@ -54,7 +54,7 @@ class StartConversationRequest(BaseModel):
     Contains an Agent configuration along with conversation-specific options.
     """
 
-    agent: AgentBase
+    agent: Agent
```

Comment:
I think we still need to keep `ActionBase`? Because we already have `Agent`

Reply:
```suggestion
    agent: AgentBase
```